seed/challenges/02-data-visualization-certification/data-visualization-certificate.json

```diff
@@ -118,7 +118,34 @@
           ""
         ]
       ],
-      "titleEs": "Reclama tu certificado de Visualización de datos"
+      "titleEs": "Reclama tu certificado de Visualización de datos",
```

Comment:
@bugron I think it's just a bug in diff or something. The postfix Fr is in line 148.